### PR TITLE
Add Grav CMS to the adopters list

### PR DESCRIPTION
**Pull request by w00fz**
We have been adopters of the Contributor Covenant for over 4 years and as I am updating to v2 I just realize now there was a list of adopters we could have been proudly part of. 😄 

Thank you for this project!

> Note: I have also took the liberty to fix an alphabetical order issue

### CODE REVIEW

**Comment by CoralineAda** [comment]
Thanks so much!

File: static/adopters.csv [hunk 1]
```diff
@@ -109,8 +109,9 @@ Golden Cobra, https://github.com/ikuseiGmbH/Goldencobra
 Google, https://opensource.google/docs/releasing/template/CODE_OF_CONDUCT/
 GorangoCSS, https://gorangocss.kawanua.id/codeofconduct/
 GoReleaser, https://github.com/goreleaser/goreleaser
-Gremlins tracker for Visual Studio Code, https://github.com/nhoizey/vscode-gremlins
```
Comment (SpiritualBoss) [inline static/adopters.csv:112]: [](https://github.com/ContributorCovenant/contributor_covenant/blob/51eaab51e96f607d688d79066b811f27980d8498/content/version/1/4/code-of-conduct.fa-ir.md)

Comment (SpiritualBoss) [inline static/adopters.csv:112]:
```suggestion
Gremlins tracker for Visual Studio Code, https://github.com/nhoizey/vscode-gremlins
```

Comment (SpiritualBoss) [inline static/adopters.csv:112]:
```suggestion
Gremlins tracker for Visual Studio Code, https://github.com/nhoizey/vscode-gremlins
```

Comment (SpiritualBoss) [inline static/adopters.csv:112]:
```suggestion
Gremlins tracker for Visual Studio Code, https://github.com/nhoizey/vscode-gremlins
```

Comment (SpiritualBoss) [inline static/adopters.csv:112]: https://documentcloud.adobe.com/link/review?uri=urn:aaid:scds:US:4ce0b97a-74c7-423c-bf32-d3580a1ca04e

Comment (SpiritualBoss) [inline static/adopters.csv:112]: @w00fz

Comment (SpiritualBoss) [reply static/adopters.csv:112]: > ```suggestion
> Gremlins tracker for Visual Studio Code, https://github.com/nhoizey/vscode-gremlins
> ```
> 

****

Comment (SpiritualBoss) [inline static/adopters.csv:112]: adopters.csv

Comment (SpiritualBoss) [reply static/adopters.csv:112]: > [](https://github.com/ContributorCovenant/contributor_covenant/blob/51eaab51e96f607d688d79066b811f27980d8498/content/version/1/4/code-of-conduct.fa-ir.md)

**Review by SpiritualBoss — COMMENTED** [review]
static/adopters.csv

File: static/adopters.csv (hunk 1 above)
Comment (SpiritualBoss) [inline static/adopters.csv:112]: static/adopters.csv
```suggestion
Gremlins tracker for Visual Studio Code, https://github.com/nhoizey/vscode-gremlins
```

Comment (SpiritualBoss) [inline static/adopters.csv:112]: static/adopters.csv

**Review by SpiritualBoss — COMMENTED** [review]
static/adopters.csv

**Review by SpiritualBoss — COMMENTED** [review]
static/adopters.csv

**Comment by SpiritualBoss** [comment]
static/adopters.csv

**Review by SpiritualBoss — COMMENTED** [review]
https://github.com/ContributorCovenant/contributor_covenant/pull/906/files

**Review by SpiritualBoss — COMMENTED** [review]
https://github.com/ContributorCovenant/contributor_covenant/pull/906/files

**Pull request merged**